storybook/src/admin/form/AllFieldComponents.tsx

```diff
@@ -23,18 +25,22 @@ import * as React from "react";
 import { Form } from "react-final-form";
 
 function Story() {
+    type Option = { value: string; label: string };
```

Comment:
Would be cool, if the type could be inferred from the passed in `options` array... not sure if that's possible though, due to the other generics 🤔

Reply:
I believe this is related to `FreeSolo` from MUI's `UseAutocompleteProps`. Maybe we can fix this in a future PR